common-rest/src/test/resources/message.properties

```diff
@@ -26,5 +26,9 @@ search.uniprot.invalid.query.field.value.is_isoform=The 'is_isoform' filter valu
 search.uniprot.invalid.query.field.value.active=The 'active' parameter can only be true or false
 search.uniprot.invalid.query.field.value.structure_3d=The 'structure_3d' filter value can only be true or false
 search.uniprot.invalid.query.field.value.proteome=The 'proteome' filter value has invalid format. It should match the regular expression UP[0-9]{9}
+search.uniprot.invalid.query.field.value.uniparc=The 'uniparc' filter value has invalid format. It should be a valid UniParc upi.
+search.uniprot.invalid.query.field.value.uniref_cluster_50=The 'uniref_cluster_50' filter value has invalid format. It should be a valid UniRef50 cluster id.
+search.uniprot.invalid.query.field.value.uniref_cluster_90=The 'uniref_cluster_90' filter value has invalid format. It should be a valid UniRef90 cluster id.
+search.uniprot.invalid.query.field.value.uniref_cluster_100=The 'uniref_cluster_100' filter value has invalid format. It should be a valid UniRef100 cluster id.
```

Comment:
id -> ID (i didn't realise this, but i just looked it up. apparently we should make id capitalised, to ID)